.bazelrc

```diff
@@ -1,7 +1,7 @@
 # Bazel distributed cache, can be temporarily disabled by passing the following
 # flag: --noremote_accept_cached
 build:darwin --remote_http_cache=https://bazel-cache.da-ext.net
-build:linux --remote_http_cache=https://bazel-cache.da-ext.net
+build:linux --remote_http_cache=https://bazel-cache.da-ext.net/ubuntu_20_04
```

Comment:
Is this enough to ensure I don't pollute our existing linux cache?

Reply:
not that worried about this tbh. I trust nix much more in this regard than Windows :)